server/admin_management/api/app/repositories/user_repository.py

```diff
@@ -36,12 +44,26 @@ def get_user_by_cognito_user_id(self, cognito_user_id: str) -> models.FamUser:
             .one_or_none()
         )
 
-    def get_users(self) -> List[models.FamUser]:
-        return self.db.query(models.FamUser).all()
+    # --- Create ---
 
     def create_user(self, fam_user: schemas.FamUserDto) -> models.FamUser:
         user_dict = fam_user.model_dump()
         db_item = models.FamUser(**user_dict)
         self.db.add(db_item)
         self.db.flush()
         return db_item
+
+    # --- Update ---
+
+    def update(self, user_id, update_values: dict, requester: str) -> int:
```

Comment:
shall we call update_user for this method maybe?

Reply:
Haha, thanks for asking. My intention was to just call it "update" when it is generic update and thinking to further refactoring few other methods (e.g., "get_user" -> "get", "create_user" -> "create", "delete_user" -> "delete" since the repository is already called **"user"_repository**) and just like common pattern to have a common service for these kind of generic functions. But then I stop, that might be too much refactoring.